doc/getting_started/index.rst

```diff
@@ -61,116 +50,84 @@ tools. First, make sure your development system OS is updated:
 Install dependencies
 ********************
 
-Next, use a package manager to install required support tools. Python 3
-and its package manager, pip, are used extensively by Zephyr for
-installing and running scripts used to compile, build, and run Zephyr
-applications.
-
-We'll also install Zephyr's multi-purpose west tool.
+Next, you'll install some host dependencies using your package manager.
 
 .. tabs::
 
    .. group-tab:: Ubuntu
 
-      #. Use the ``apt`` package manager to install these tools:
+      #. Use ``apt`` to install dependencies:
 
          .. code-block:: bash
 
-            sudo apt install --no-install-recommends git cmake ninja-build gperf \
-              ccache dfu-util device-tree-compiler wget \
-              python3-pip python3-setuptools python3-tk python3-wheel xz-utils file \
-              make gcc gcc-multilib g++-multilib libsdl2-dev
+            sudo apt install --no-install-recommends cmake ninja-build gperf \
+              python3 python3-setuptools python3-pip python3-wheel git \
+              ccache wget xz-utils file
```

Comment:
native posix won't work if you don't get the multilibs. Are you sure we want to remove those?

Reply:
How about we move those to the toolchain page in that case?

Reply:
I think the getting started guide should get you all needed tools without looking elsewhere.

And being able to download Zephyr and just test it by building native posix, is one such thing that should be possible out-of-the box.

Just as it makes life easier for Linux users, that Zephyr provides a SDK with toolchains, so you don't need to go to 3rd party toolchain page (which you need to do on windows)

Reply:
> How about we move those to the toolchain page in that case?

how is this related to toolchains? Agree with @tejlmand

Reply:
Those are the toolchains that are needed to build a particular target (native_posix)... what am I missing?

Reply:
and so is: `arm`, `arc`, `xtensa`, and all the other toolchains.
Why should they work, and not `native_posix`, when we don't know which toolchain the users will be using with Zephyr ?

Reply:
Agreed, I suggest leaving the multilib requirements as they were. native_posix is very important to many newcomers to Zephyr.

Reply:
I've restored the Ubuntu packages and moved the Python script changes to their own PR, https://github.com/zephyrproject-rtos/zephyr/pull/24248